src/Filter/Filter.php

```diff
@@ -263,13 +264,22 @@ public function addAttribute($name, $value)
 	/**
 	 * @return array
 	 */
-	public function getAttribtues()
+	public function getAttributes()
 	{
 		return $this->attributes;
 	}
 
 
 	/**
+	 * @return \Ublaboo\DataGrid\Column\Column
+	 */
+	public function getColumnObject()
```

Comment:
Yeah I know, that would be kinda not to break the chain :) I found it usefull in my project but I actually thought it's going to be rejected...